src/mlpack/tests/sgdr_test.cpp

```diff
@@ -43,7 +43,8 @@ BOOST_AUTO_TEST_CASE(CyclicalResetTest)
     {
       double epochStepSize = stepSize;
 
-      CyclicalDecay cyclicalDecay(restart, double(mult), stepSize, 10, 1000);
+      CyclicalDecay cyclicalDecay(restart, double(mult), stepSize);
+      cyclicalDecay.EpochBatches() = (double)1000 / 10;
```

Comment:
Can you add a space between the cast and the argument?  For this one you could also just write `100.0`. :)